hbase-it/src/test/java/org/apache/hadoop/hbase/HBaseClusterManager.java

```diff
@@ -322,7 +323,10 @@ protected CommandProvider getCommandProvider(ServiceType service) throws IOExcep
       case ZOOKEEPER_SERVER:
         return new ZookeeperShellCommandProvider(getConf());
       default:
-        return new HBaseShellCommandProvider(getConf());
+        Class<? extends CommandProvider> provider = getConf()
+          .getClass("hbase.it.clustermanager.hbase.command.provider",
```

Comment:
[not a blocker][no action required] I found many inline property keys in this class, but you don't need to change it 
now. but at some point in the future, we may need to move all these `hbase.it.clustermanager.*` to a single place, e.g. `HBaseItClusterMangerConstants`